README.md

```diff
@@ -11,13 +11,15 @@
 
 [//]: # (One or two sentences description of the module/service in non technical terms and information about the owner. Text in quotes are examples.)
 
-Game room management service, designed to manage multiple game room fleets in isolated schedulers.
+Dedicated game server management service, designed to manage multiple game servers fleets in isolated schedulers.
```

Comment:
The initial description mentions game room without defining what it is first, so I used correct term here